README.md

```diff
@@ -32,11 +32,31 @@ dotnet tool install Cake.Tool -g
 #### IDE
 - If you have Visual Studio 16.9 P4 installed you can open `Maui.sln` and run it from there.
 
-#### Command Line
+#### .NET 6
 
+You can run a `Cake` target to bootstrap .NET 6 in `bin\dotnet` and launch Visual Studio:
+
+```dotnetcli
+dotnet cake --target=VS-NET6
+```
+_NOTE: VS Mac is not yet supported._
+
+You can also run commands individually:
+```dotnetcli
+# Provision .NET 6 in bin\dotnet
+dotnet build src\DotNet\DotNet.csproj
+# Builds Maui MSBuild tasks
+.\bin\dotnet\dotnet build Microsoft.Maui.BuildTasks-net6.sln
+# Builds the rest of Maui
+.\bin\dotnet\dotnet build Microsoft.Maui-net6.sln
+# (Windows-only) to launch Visual Studio
+.\eng\dogfood.ps1
 ```
-dotnet build src\Controls\samples\Controls.Sample.Droid\Maui.Controls.Sample.Droid-net6.csproj -t:Run
-dotnet build src\Controls\samples\Controls.Sample.iOS\Maui.Controls.Sample.iOS-net6.csproj -t:Run
+
+To build & run .NET 6 sample apps, you will also need to use `.\bin\dotnet\dotnet`:
+```dotnetcli
+.\bin\dotnet\dotnet build src\Controls\samples\Controls.Sample.Droid\Maui.Controls.Sample.Droid-net6.csproj -t:Run
+.\bin\dotnet\dotnet build src\Controls\samples\Controls.Sample.iOS\Maui.Controls.Sample.iOS-net6.csproj -t:Run
```

Comment:
This describes the new workload for .NET 6.

I made the existing `Cake` target work: `dotnet cake --target=VS-NET6`

Reply:
maybe a reference to the --no-restore?

Reply:
I added another example of the `SingleProject` one 👍